var/spack/repos/builtin/packages/chapel/package.py

```diff
@@ -3,24 +3,784 @@
+        default="unset",
+        multi=False,
+        description="Selects between no jemalloc, bundled jemalloc, or system jemalloc",
+    )
+
+    variant(
+        "host_mem",
+        values=("cstdlib", "jemalloc"),
+        default="jemalloc",
+        description="Memory management layer for the chpl compiler",
+        multi=False,
+    )
+
+    variant(
+        "host_platform",
+        description="Host platform",
+        default="unset",
+        values=platform_opts,
+        multi=False,
+    )
+
+    variant(
+        "hwloc",
+        description="Build with hwloc support",
+        default="bundled",
+        values=("bundled", "none"),
+        multi=False,
+    )
+
+    variant(
+        "launcher",
+        values=launcher_names,
+        default="unset",
+        description="Launcher to use for running Chapel programs",
+        multi=False,
+    )
+
+    variant(
+        "lib_pic",
+        values=("none", "pic"),
+        default="none",
+        description="Build position-independent code suitable for shared libraries",
+    )
+
+    variant(
+        "libfabric",
+        default="unset",
+        description="When building with ofi support, specify libfabric option",
+        values=("bundled", "system", "unset"),
+        multi=False,
+    )
+
+    variant(
+        "llvm",
+        default="spack",
+        description="LLVM backend type. Use value 'spack' to have spack "
+        "handle the LLVM package",
+        values=("bundled", "none", "spack", "system"),
+    )
+
+    variant(
+        "package_modules",
+        description="Include package module dependencies with spack",
+        values=disjoint_sets(("none",), ("all",), package_module_dict.keys())
+        .with_error("'none' or 'all' cannot be activated along with other package_modules")
+        .with_default("all")
+        .with_non_feature_values("none", "all"),
+    )
+
+    variant(
+        "re2",
+        description="Build with re2 support",
+        default="bundled",
+        values=("bundled", "none"),
+        multi=False,
+    )
+
+    variant(
+        "target_arch",
+        description="Target architecture for cross compilation",
+        default="unset",
+        values=("x86_64", "aarch64", "arm64", "unset"),
+        multi=False,
+    )
+
+    # Feedback that it's hard to imagine any circumstance where host_compiler or
+    # target_compiler could meaningfully differ from what Spack already knows about
+    # variant(
+    #     "target_compiler",
+    #     values=compilers,
+    #     description="Compiler suite for building runtime libraries and "
+    #     "generated code on CHPL_TARGET_PLATFORM",
+    #     default="unset",
+    # )
+
+    variant(
+        "target_cpu",
+        values=cpu_options,
+        description="Indicate that the target executable should be specialized "
+        "to the given architecture when using --specialize (and --fast).",
+        default="unset",
+        multi=False,
+    )
```

Comment:
this is another value that is just used inside the chapel build scripts, but it is important to specify when installing because of the heterogenous nature of some machines